.github/pr-title-checker-config.json

```diff
@@ -0,0 +1,15 @@
+{
+  "LABEL": {
+    "name": ""
+  },
+  "CHECKS": {
+    "NOTE": "You can test the regex here: https://regex101.com/r/W2f7Yr/1",
+    "regexp": "((?<type>build|chore|ci|docs|feat|fix|perf|refactor|revert|style|test|BREAKING CHANGE)(?<scope>\\(\\w+\\))?(?<breaking_change>!?): (?<summary>[\\w -]+))(?<body>[\\w\\s-]+)(?<footer>(?<=\\v\\v)(?<footer_token>[\\w-]+): (?<footer_value>[\\w -]+)|$)",
```

Comment:
Can we simplify the regex? Currently, it allows only words in the body. What are your thoughts on using this instead?
`((?<type>build|chore|ci|docs|feat|fix|perf|refactor|revert|style|test|BREAKING CHANGE)(?<scope>\(\w+\))?(?<breaking_change>!?): (?<body>.+)$)`

Reply:
Sure maybe this is a good idea.